src/commands/copy/README.md

```diff
@@ -0,0 +1,32 @@
+# copy
+
+Copy a manifest of files
+
+## Usage
+
+copy <options> [...location]
+
+### Arguments
+
+| Usage         | Description              | Options |
+| ------------- | ------------------------ | ------- |
+| [...location] | Manifest of file to copy |         |
```

Comment:
`[...foo]` is not a syntax I'm familiar with. What's meant here? Is it zero-or-one locations? If so, `[location]` is more common syntax for that. If it's exactly one location, that would normally be `<location>`.

Reply:
this is just the output of the help text, I would imagine it means `string[]` as `...` is the javascript spread operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
So we want to use that syntax? OK.